webapp/channels/src/components/convert_gm_to_channel_modal/no_common_teams.tsx

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+import {useIntl} from 'react-intl';
+
+import SectionNotice from 'components/section_notice';
+
+const NoCommonTeamsError = (): JSX.Element => {
```

Comment:
Any particular reason this component was moved from `no_common_teams/no_common_teams.tsx` to one level up?